files/scripts/syncd.sh

```diff
@@ -164,11 +164,20 @@ stop() {
         debug "${TYPE} shutdown syncd process ..."
         /usr/bin/docker exec -i syncd$DEV /usr/bin/syncd_request_shutdown --${TYPE}
 
-        # wait until syncd quits gracefully
-        while docker top syncd$DEV | grep -q /usr/bin/syncd; do
+        # wait until syncd quits gracefully or force syncd to exit after 
+        # waiting for 20 seconds
+        start_in_secs=`date +%s`
+        end_in_secs=`date +%s`
+        while docker top syncd$DEV | grep -q /usr/bin/syncd \
+                && [[ $((end_in_secs - start_in_secs)) -le 20 ]]; do
```

Comment:
Suggest defining a constant for the timeout value

Reply:
Great suggestion! Updated.